src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -302,6 +302,16 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
                     runtimeConfig.AddService(iSetup.TimeZoneInfo);
                 }
 
+                // Ensure tests can run with governor on. 
+                // Some tests that use large memory can disable via:
+                //    #SETUP: DisableMemChecks
+                if (!iSetup.DisableMemChecks)
+                {
+                    var kbytes = 1000;
+                    var mem = new BasicGovernor(10 * 1000 * kbytes);
+                    runtimeConfig.AddService<Governor>(mem);
```

Comment:
Ensure that all our .txts run with this on. 
A few tests do Sequence(100000) and that blows the limit, hence the #setup to disable it.